addons/medical/functions/fnc_addDamageToUnit.sqf

```diff
@@ -29,7 +29,7 @@ params [
     ["_bodyPart", "", [""]],
     ["_typeOfDamage", "", [""]],
     ["_instigator", objNull, [objNull]],
-    ["_damageSelectionArray", [], [[]]],
+    "_",
```

Comment:
This isn't a neat solution to the parameter issue, but so far I can't think of anything better.

Reply:
You can use empty string to denote unused parameter.